python/dask_cudf/dask_cudf/expr/_collection.py

```diff
@@ -113,7 +130,7 @@ def struct(self):
         return StructMethods(self)
 
 
-class Index(DXIndex):
+class Index(CudfFrameBase, DXIndex):
```

Comment:
Think the ordering of the parent classes here matters, as we want `dask_expr.Index` overrides to take preference over our `CudfFrameBase` overrides - think this applies for all the classes but is important in particular here because it looks like `dask_expr.Index.var` isn't implemented:

```suggestion
class Index(DXIndex, CudfFrameBase):
```

Reply:
Okay, I think you're right.